### PR TITLE
Added body pillows

### PR DESCRIPTION
**Pull request by FulcrumA**
#### Summary
SUMMARY: Content "Added bodypillows"  

#### Purpose of change
Adding a variation on pillows, due to popular demand.

#### Describe the solution
A pillow was added.

#### Describe alternatives you've considered
Not adding it and berating people who asked me to do this for being silly.

#### Additional context
Powers that be, have mercy on us all.

### CODE REVIEW

**Comment by Soyweiser** [comment]
Might want to add it to droplists, if you want to see it ingame.

File: data/json/item_groups.json [hunk 1]
```diff
@@ -5642,6 +5644,7 @@
       ["hygrometer", 3],
       ["barometer", 3],
       ["dnd_handbook", 2]
+      ["bodypillow", 1]
```
Comment (Vzor-) [inline data/json/item_groups.json]: Missing a ',' here I think

Comment (FulcrumA) [reply data/json/item_groups.json]: That is correct. Thank you.

**Comment by Mecares** [comment]
Damn it, now I have to mod in a profession that starts with a body pillow and a fake katana.

**Comment by Kelenius** [comment]
>Damn it, now I have to mod in a profession that starts with a body pillow and a fake katana.

Don't forget a fedora.

**Comment by kevingranade** [comment]
>  >   Damn it, now I have to mod in a profession that starts with a body pillow and a fake katana.

> Don't forget a fedora.

M'zombie

**Comment by Mecares** [comment]
Dont forget;

"use_action":[
    "msg" : "You hug your waifu/husbando.", or   "msg" : "Look we do not judge or anything, but this is starting to get a bit weird even by our standards.",

**Comment by nexusmrsep** [comment]
Shouldn't it be rather a Crazy Cataclysm Mod item?

**Comment by FulcrumA** [comment]
@nexusmrsep 
Eh, such kind of item is not more crazy than current fursuits. It's a thing that indeed is available IRL. If we'd go with suggestion of @Mecares and add fedora-tipping neckbeard as a "profession" then we'd be getting somewhat further into the weirdness territory but body pillow is just a regular, albeit quirky and sometimes questionable IRL item. It may be also rare, though, at least in New England - which thing I've taken into account while adding it to the loot list.

**Comment by nexusmrsep** [comment]
Don't get me wrong, I'm not against it in any way, just loudly considering the options. Your explanation convinces me.

**Comment by FulcrumA** [comment]
Sure, you've raised a valid concern and I just addressed it. No assumptions! :)

**Comment by Soyweiser** [comment]
Craft and disassemble recipes? Or only disassemble I guess. (As crafting the print is not really possible during the apocalypse).

**Comment by FulcrumA** [comment]
@Soyweiser
No need to specify such, IMHO as it can be already cut into rags just like any other pillow.

**Comment by Xhuis** [comment]
I think this should probably be Crazy Cataclysm-specific.

**Comment by Rivet-the-Zombie** [comment]
I don't see what's so 'crazy' about adding this to the base game; people actually own these things in real life.

**Comment by Xhuis** [comment]
People also own CP. Something existing in real life is not an argument to add it, to me. But, I'm probably the sole voice of dissent here.

**Comment by Soyweiser** [comment]
I thought the crazy mod was more for crazy stuff that isn't really 'real' like the dancer zombie summoning hulks, and the crazy gamebreaking statshuffeling granade. And other over the top not-real references  (smoky the bear, shia the actual cannibal).

The body pillows (no matter how cringy you might think addition of them is) isn't something that should be in the crazy cataclysm mod. It is just some object that some people find cringy to include (like fursuits, or vibrators, or bdsm gear, or whatever).

Which is utterly different from childporn, so a bit of a false equivalence there.

**Comment by overbo** [comment]
They shouldn't be super rare and shouldn't be questionable IRL.  They are a somewhat common item to ease the discomfort of sleeping while massively pregnant.

Maybe a waifu / husbando pillowcase as a container item for the body pillow, and the container could have the aforementioned use_action.

**Comment by DracoGriffin** [comment]
>I agree that this should probably be Crazy Cataclysm-specific. This feels vastly out-of-place. I believe that the fur suits predate Crazy Cataclysm, so it makes sense that they're not in there. Grandfather clause isn't proper justification, in my eyes.
I left the Discord because of stuff like this; I don't want it to leak into the game, too.

I agree. The description used currently "scantily clad" could be a much more rare version and/or profession-specific or confined to Crazy Cataclysm, but body pillow itself should be fine.

As for reference, when I hear "[body pillow](https://www.amazon.com/body-pillows/b?ie=UTF8&node=3731991)", I usually think of the long pillows that are cuddled, mainly for pregnant women.

**Comment by Xhuis** [comment]
Yes, this isn't a body pillow; this is a daki. I'd be fine with the addition if the full-size pillow was added as a sleeping aid and a rare version could spawn as a daki, since then it'd be an easter egg variant of an existing item with an actual purpose rather than just one put in 4 teh lulz.

**Comment by Xhuis** [comment]
I'm of the opinion that the suits should be removed (no joke, I set the building on fire whenever I find one - I don't hate it *that* much but it's a funny mental image), and that faux fur should get more clothing items. Actually, I should get on that. In any case, that's my opinion, not law; the line is drawn there, even though I disapprove of it. Ultimately, it's your PR, so it's your choice unless a maintainer intercedes.

**Comment by Theundyingcode** [comment]
I along with a number of people at college had/have a body pillow to place on our beds against the wall like couch backings (we used our beds as couches).  The thing is they are all either plain or patterned not waifu print.  I see no problem at all with body pillows themselves being in main.  That said moving the waifu comments to crazy cataclysm is valid.

**Comment by Xhuis** [comment]
I'm disappointed that this got in at this current state, but as the sole voice of dissent in an otherwise-content PR, I suppose it's not my place to give more negative input.

**Comment by DracoGriffin** [comment]
@Xhuis You're not the only one, there were others that agreed in part, myself included.

**Pull request merged**